### PR TITLE
Finished

### PR DESCRIPTION
**Pull request by bhambleton123**
Think I'm ready to start on proxy stuff.

### CODE REVIEW

File: client/components/app.jsx [hunk 1]
```diff
@@ -1,42 +1,116 @@
...
+      <div className="app-style">
+        <Cost initial={this.state.cost} />
+        <Line />
+        <Dates unavailableDates={this.state.unavailableDates} checkIn={this.state.checkIn} checkOut={this.state.checkOut} getDate={this.getDate}/>
+        <Guests />
         <Line/>
-        <Dates/>
-        <Guests/>
-        <ReserveButton/>
+        {this.state.checkIn !== '' && this.state.checkOut !== '' ? <Info total={this.state.total} setTotal={this.setTotal} initial={this.state.cost} checkIn={this.state.checkIn} checkOut={this.state.checkOut} cleaning={this.state.cleaning} service={this.state.service} taxes={this.state.taxes}/> : ''}
```
Comment (RachNovo) [inline client/components/app.jsx]: On mine when a component had a lot of variables I laid them out on new lines so the line isn't so long @bhambleton123

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+DB_HOST = localhost;
```
Comment (therobinkim) [inline .env]: `.env` files should generally not be committed

**Comment by therobinkim** [comment]
FYI I will not be commenting on things that the Airbnb style guide already addresses. (You should install ESLint w/ Airbnb configs asap!)

File: server/models/index.js [hunk 3]
```diff
@@ -41,8 +44,8 @@ let getUsers = (callback) => {
 //right now, we're just posting the dates that are unavailable when someone books a house since there's currently no authentication
 
 
-let saveHouse = (name, initialPrice, unavailableDates, pricePerGuest) => {
-  const hotel = new Hotel({name: name, initialPrice: initialPrice, unavailable_dates: unavailableDates, price_per_guest: pricePerGuest});
+let saveHouse = (name, initialPrice, cleaning, service, taxes, unavailableDates, pricePerGuest) => {
+  const hotel = new Hotel({name: name, initialPrice: initialPrice, cleaning: cleaning, service: service, taxes: taxes, unavailable_dates: unavailableDates, price_per_guest: pricePerGuest});
```
Comment (therobinkim) [inline server/models/index.js]:
```
{
  name,
  initialPrice,
  cleaning,
  service,
  taxes,
  unavailable_dates: unavailableDates,
  price_per_guest: pricePerGuest
}
```

File: server/index.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ app.get('/api/users', (req, res) => {
 })
 
 app.post('/api/houses', (req, res) => {
-  models.saveHouse(req.body.name, req.body.initialPrice, req.body.availableDates, req.body.pricePerGuest);
+  models.saveHouse(req.body.name, req.body.initialPrice, req.body.cleaning, req.body.service, req.body.taxes, req.body.availableDates, req.body.pricePerGuest);
   res.end("Worked");
```
Comment (therobinkim) [inline server/index.js]: You seem to be ending the response with `Worked` whether the `models.saveHouse` operation actually worked or not.

File: server/index.js [hunk 5]
```diff
@@ -43,7 +43,7 @@ app.get('/api/users', (req, res) => {
 })
 
 app.post('/api/houses', (req, res) => {
-  models.saveHouse(req.body.name, req.body.initialPrice, req.body.availableDates, req.body.pricePerGuest);
+  models.saveHouse(req.body.name, req.body.initialPrice, req.body.cleaning, req.body.service, req.body.taxes, req.body.availableDates, req.body.pricePerGuest);
```
Comment (therobinkim) [inline server/index.js]: Requiring devs to remember the exact order of _seven_ arguments is a lot!
Rather, combine the required args into a single object.

```
models.saveHouse({
  name,
  initialPrice,
  cleaning,
  service,
  taxes,
  availableDates,
  pricePerGuest
}, callback);
```

File: client/components/line.jsx [hunk 6]
```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import styles from './styles.js';
+import './css/app.css';
 
 const Line = (props) => {
-  return <hr style={styles.line}></hr>;
```
Comment (therobinkim) [inline client/components/line.jsx]: I think you should've left line 5 alone; I would recommend reverting it (along w/ a lot of the `className` changes you made)

File: client/components/guests.jsx [hunk 7]
```diff
@@ -1,21 +1,100 @@
...
+  increaseAdults() {
+    this.setState({
+      adults: this.state.adults+1
+    })
+  }
+
+  decreaseAdults() {
+    if(this.state.adults > 1){
+      this.setState({
+        adults: this.state.adults-1
```
Comment (therobinkim) [inline client/components/guests.jsx]: You should use the callback form of `this.setState` if you are updating the state based on its current/previous version. https://reactjs.org/docs/react-component.html#setstate

File: client/components/calendar/calendar.jsx [hunk 8]
```diff
@@ -91,23 +91,24 @@ class Calendar extends Component {
...
+          <button className="button-right" onClick={this.dateRight}>→</button>
+        </div>
+        <table className="table">
           <tbody>
             <tr>
-              {weeksArray.map((day, i) => <td key={i} style={Styles.calendarText}>{day}</td>)}
+              {weeksArray.map((day, i) => <td key={i} className="calendar-text">{day}</td>)}
             </tr>
-            {this.state.weeks.map((week, i) => <Week key={i} dates={week} incrementMonth={this.state.incrementMonth}/>)}
+            {this.state.weeks.map((week, i) => <Week unavailableDates={this.props.unavailableDates} checkIn={this.props.checkIn} checkOut={this.props.checkOut} calType={this.props.calType} getDate={this.props.getDate} key={i} dates={week} incrementMonth={this.state.incrementMonth} />)}
```
Comment (therobinkim) [inline client/components/calendar/calendar.jsx]: Avoid cramming so much code into one line. A linter can help you identify these instances. https://github.com/airbnb/javascript/tree/master/react#alignment

**Pull request merged**